File: app/src/main/java/mineproject/com/notes/NavigationActivity.java
package mineproject.com.notes;

import android.os.Bundle;
import com.google.android.material.bottomnavigation.BottomNavigationView;
import androidx.appcompat.app.AppCompatActivity;
import androidx.annotation.NonNull;
import android.view.MenuItem;
import android.widget.TextView;

public class NavigationActivity extends AppCompatActivity {
    private TextView mTextMessage;

    private BottomNavigationView.OnNavigationItemSelectedListener mOnNavigationItemSelectedListener
            = new BottomNavigationView.OnNavigationItemSelectedListener() {

        @Override
        public boolean onNavigationItemSelected(@NonNull MenuItem item) {
            switch (item.getItemId()) {
                case R.id.navigation_notes:
                    mTextMessage.setText(R.string.title_notes);
                    return true;
                case R.id.navigation_tasks:
                    mTextMessage.setText(R.string.title_tasks);
                    return true;

            }
            return false;
        }
    };

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_navigation);
        BottomNavigationView navView = findViewById(R.id.nav_view);
        mTextMessage = findViewById(R.id.message);
        navView.setOnNavigationItemSelectedListener(mOnNavigationItemSelectedListener);

        Tag tag = new Tag(
                "High Priority",
                R.color.colorAccent
        );

        Task task = new Task(
                "Get Food"
        );

        String s = task.getTitle();
    }

}

File: app/src/main/java/mineproject/com/notes/navigation/NavigationActivity.kt
package mineproject.com.notes.navigation

import android.os.Bundle
import com.google.android.material.bottomnavigation.BottomNavigationView
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_navigation.*
import mineproject.com.notes.R

class NavigationActivity : AppCompatActivity() {

    private val mOnNavigationItemSelectedListener = BottomNavigationView.OnNavigationItemSelectedListener { item ->
        when (item.itemId) {
            R.id.navigation_notes -> {
                messageTextView.setText(R.string.title_notes)
                return@OnNavigationItemSelectedListener true
            }
            R.id.navigation_tasks -> {
                messageTextView.setText(R.string.title_tasks)
                return@OnNavigationItemSelectedListener true
            }
        }
        false
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_navigation)

        messageTextView.text = getString(R.string.title_tasks)
        navigationView.setOnNavigationItemSelectedListener(mOnNavigationItemSelectedListener)
    }

}
